.github/workflows/hardware-smoke.yml

```diff
@@ -158,6 +163,11 @@ jobs:
         with:
           app-path: tt-zephyr-platforms
 
+      - uses: ./tt-zephyr-platforms/.github/workflows/prepare-hardware
+        with:
+          board: ${{ matrix.config.board }}
+          version: '80.15.0.0'
```

Comment:
Same here - 80.18.0.0 would probably be a bit more stable, but maybe I'm wrong.

Reply:
It probably is, this PR was just created a while ago. Maybe instead of storing the FW bundles locally, we do something like the following:
- flash an old (but stable) copy of BMFW via the debug connection
- use `tt-flash` to flash the latest stable FW bundle from the tt-firmware repo

Does that seem reasonable?

Reply:
Yes, I think that would make sense.

In the mean time, I'll approve. We can merge this if you prefer, and then you could work on pulling stable firmware from the repository in a separate PR, or you can update this one, depending on what is easier for you.